public/locales/en/resource.json

```diff
@@ -120,6 +120,8 @@
     "grades.passable": "Passable",
     "grades.inadequate": "Inadequate",
     "grades.mediocre": "Mediocre",
+    "admin.revert-custom-grades": "Revert to default grades",
+    "admin.revert-custom-grades-desc": "Reverting to default grades will delete all your custom grades (Press toggle on won't make them reappear).",
```

Comment:
Maybe: "Reverting to default grades will delete all your current custom grades. (Reactivating custom grades won't make them reappear.)"